src/scripts/update-firmware-binaries.js

```diff
@@ -234,20 +234,24 @@ const downloadAppBinaries = async (): Promise<*> => {
     repo: GITHUB_FIRMWARE_REPOSITORY,
   });
 
-  const versionText = new Buffer(versionResponse.content, 'base64').toString();
-  const startIndex = versionText.indexOf('| 0 ');
-  const endIndex = versionText.indexOf('\n\n', startIndex);
+  const versionText =
+    new Buffer(versionResponse.content, 'base64').toString() || '';
   const data = versionText
-    .substring(startIndex, endIndex)
-    .replace(/\s/g, '')
-    .split('|');
-
+    .match(/^\|[^\n]*/gim)
```

Comment:
I guess this adds one more flow error, `match` can return undefined, if flow complains you have to wrap it into `nullthrows`